modules/reference/pages/properties/object-storage-properties.adoc

```diff
@@ -333,7 +343,15 @@ Maximum size of object storage cache. If both this property and <<cloud_storage_
 
 === cloud_storage_cache_size_percent
 
-Maximum size of the cloud cache as a percentage of unreserved disk space disk_reservation_percent. The default value for this option is tuned for a shared disk configuration. Consider increasing the value if using a dedicated cache disk. The property <<cloud_storage_cache_size,`cloud_storage_cache_size`>> controls the same limit expressed as a fixed number of bytes. If both `cloud_storage_cache_size` and `cloud_storage_cache_size_percent` are set, Redpanda uses the minimum of the two.
+Maximum size of the cache as a percentage, minus the space that Redpanda avoids using defined by the xref:reference:cluster-properties.adoc#disk_reservation_percent[`disk_reservation_percent`] cluster property. This is calculated at startup and dynamically updated if either this property or  <<cloud_storage_cache_size,`cloud_storage_cache_size`>> changes.
```

Comment:
>if either this property or <<cloud_storage_cache_size,`cloud_storage_cache_size`>> changes.

Should be reflect that it is also updated if `disk_reservation_percent` is changed.

Reply:
```suggestion
Maximum size of the cache as a percentage, minus the space that Redpanda avoids using defined by the xref:reference:cluster-properties.adoc#disk_reservation_percent[`disk_reservation_percent`] cluster property. This is calculated at startup and dynamically updated if either this property, `disk_reservation_percent` or  <<cloud_storage_cache_size,`cloud_storage_cache_size`>> changes.
```